File: src/main/java/br/com/leonardoferreira/jirareport/config/DefaultAcceptClient.java
package br.com.leonardoferreira.jirareport.config;

import feign.RequestInterceptor;
import feign.RequestTemplate;
import org.springframework.stereotype.Component;

@Component
public class DefaultAcceptClient implements RequestInterceptor {

    @Override
    public void apply(final RequestTemplate template) {
        if (template != null) {
            template.header("Accept", "application/json");
        }
    }

}

File: src/main/java/br/com/jiratorio/config/DefaultAcceptClient.kt
package br.com.jiratorio.config

import feign.RequestInterceptor
import feign.RequestTemplate
import org.springframework.stereotype.Component

@Component
class DefaultAcceptClient : RequestInterceptor {

    override fun apply(template: RequestTemplate?) {
        template?.header("Accept", "application/json")
    }

}
